sympy/functions/special/zeta_functions.py

```diff
@@ -384,7 +385,11 @@ def _eval_nseries(self, x, n, logx, cdir=0):
                     s.append(term/k**nu)
                 return Add(*s) + o
 
-        return super(polylog, self)._eval_nseries(x, n, logx, cdir)
+        result = super(polylog, self.expand())._eval_nseries(x, n, logx, cdir)
+        if ((1-nu).is_integer and (1-nu).is_nonpositive):
+            return result
+        result += gamma(1-nu)*(((log(1/z))**(nu-1)).nseries(x, n=n, logx=logx))
+        return result
```

Comment:
Three-valued logic needs to be handled here. Is it correct to add this term to `result` if `(1 - nu).is_integer` gives `None` as it would for a generic symbol?

https://docs.sympy.org/latest/guides/booleans.html

Also `(1 - nu).is_integer` should just be `nu.is_integer`.

Reply:
Hey @arnabnandikgp could you address this ?